www/templates/html/Peoplefinder/Instructions.tpl.php

```diff
@@ -61,18 +61,18 @@
               	<img class="hero-img" src="data:image/gif;base64,R0lGODlhAQABAAAAADs=" alt="Exterior view of Canfield Administration Building on a sunny day.">
             </picture>
             <div class="dcf-card-block">
-                <h2 class="dcf-txt-h3"><span class="dcf-d-block dcf-subhead unl-dark-gray dcf-mb-1">Search </span>Departments</h2>
-                <p class="dcf-txt-sm">Find departments by entering a full or partial department name. Information available:</p>
+                <h2 class="dcf-txt-h3"><span class="dcf-d-block dcf-subhead unl-dark-gray dcf-mb-1">Search </span>Departments &amp; Units</h2>
+                <p class="dcf-txt-sm">Find departments &amp; units by entering a full or partial name. Information available:</p>
```

Comment:
I think, in the context of a sentence, we should spell out "and" instead of using an ampersand.